doc/releases/changelog-dev.md

```diff
@@ -291,6 +291,10 @@ the `decimals` and `show_matrices` keywords are added. `qml.drawer.tape_text(tap
 
 <h3>Bug fixes</h3>
 
+* Fixes a bug where the default `parameter_frequencies` of single-parameter gates
+  were formatted wrongly.
+  [(#2383)](https://github.com/PennyLaneAI/pennylane/pull/2383)
```

Comment:
:100: for the fix, but you may also want to explain the addition of the checker in the changelog?

Reply:
:facepalm: